automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -1,274 +1,180 @@
 package org.broadinstitute.dsde.firecloud.page.workspaces.methodconfigs
 
+import org.broadinstitute.dsde.firecloud.component._
 import org.broadinstitute.dsde.firecloud.config.Config
-import org.broadinstitute.dsde.firecloud.page.components.Table
 import org.broadinstitute.dsde.firecloud.page.workspaces.WorkspacePage
 import org.broadinstitute.dsde.firecloud.page.workspaces.monitor.SubmissionDetailsPage
-import org.broadinstitute.dsde.firecloud.page.{ErrorModal, FireCloudView, PageUtil}
-import org.openqa.selenium.{JavascriptExecutor, WebDriver}
+import org.broadinstitute.dsde.firecloud.page.{ErrorModal, OKCancelModal, PageUtil}
+import org.openqa.selenium.WebDriver
 import org.scalatest.selenium.Page
 
 class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodConfigNamespace: String, methodConfigName: String)(implicit webDriver: WebDriver)
   extends WorkspacePage(namespace, name) with Page with PageUtil[WorkspaceMethodConfigDetailsPage] {
 
-  override val url: String = s"${Config.FireCloud.baseUrl}#workspaces/$namespace/$name/method-configs/$methodConfigNamespace/$methodConfigName"
-
-  def launchAnalysis(rootEntityType: String, entityId: String, expression: String = "", enableCallCaching: Boolean = true) = {
-    val launchModal = ui.openLaunchAnalysisModal()
-    launchModal.launchAnalysis(rootEntityType, entityId, expression, enableCallCaching)
-    new SubmissionDetailsPage(namespace, name)
+  override def awaitReady(): Unit = {
+    await condition isLoaded
+    await spinner "Checking permissions..."
```

Comment:
wondering why we call isLoaded instead of moving what it checks for to here...

Reply:
isLoaded is used in a few places. I don't necessarily like that it is, but I didn't want to change unit tests for this (besides cleaning up how they wait for things to be ready).